data.sql

```diff
@@ -1,6 +1,42 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (01, 'Augmon', '2020-02-03', 0, true, 10.23);
-INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (02, 'Gabumon', '2018-11-15', 2, true, 8.0);
-INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (03, 'Pikachu', '2021-01-03', 1, false, 15.04);
-INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (04, 'Devimon', '2017-03-12', 5, true, 11.0);
+INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (01, 'Charmander', '2020-02-08', 0, false, -11.0);
+INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (02, 'Plantmon', '2021-02-08', 2, true, -5.7);
+INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (03, 'Squirtle', '1993-04-02', 3, false, -12.13);
+INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (04, 'Angemon', '2005-06-12', 1, true, -45.0);
+INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (05, 'Boarmon', '2005-06-07', 7, true, 20.4);
+INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (06, 'Blossom', '1998-10-13', 3, true, 17.0);
+INSERT INTO animals(ID, NAME, DOB, ESCAPE_ATTEMPTS, NEUTERED, WEIGHT_KG) VALUES (07, 'Ditto', '2022-05-14', 4, true, 22.0);
```

Comment:
- Optional
I would recommend generating the Id automatically instead of entering it manually. Imagine a scenario where you have to enter 100's of animals in different transactions. That would be difficult. Right? And you will also have to check the database for the last ID of the animal and then proceed from there.